SentinelOne/sentinelone/ingest/parser.yml

```diff
@@ -299,6 +297,22 @@ stages:
           sentinelone.data.registrypath: "{{json_event.message.data.registrypath}}"
           sentinelone.data.registryvalue: "{{json_event.message.data.registryvalue}}"
           sentinelone.data.ruleId: "{{json_event.message.data.ruleId}}"
+          sentinelone.data.ruleid: "{{json_event.message.data.ruleid}}"
+          rule.id: "{{json_event.message.data.ruleid or json_event.message.data.ruleId}}"
+          sentinelone.data.confidenceLevel: "{{json_event.message.data.confidenceLevel}}"
+          sentinelone.data.escapedMaliciousProcessArguments: "{{json_event.message.data.escapedMaliciousProcessArguments}}"
+          sentinelone.data.fileContentHash: "{{json_event.message.data.fileContentHash}}"
```

Comment:
Do you know the digest method used for the hash?
I think you could use `file.hash.<digest method>` instead.

Reply:
I don't know .. =/

Could be SHA-1